src/components/wordchain/WordchainWinners/index.tsx

```diff
@@ -0,0 +1,111 @@
+import styled from '@emotion/styled';
+
+import WordChainWinner from '@/components/wordchain/WordchainWinners/WordChainWinner';
+import { colors } from '@/styles/colors';
+import { MOBILE_MEDIA_QUERY } from '@/styles/mediaQuery';
+import { textStyles } from '@/styles/typography';
+
+export default function WordchainWinners() {
+  const WORDCHAIN_WINNERS_DATA = {
+    winners: [
+      {
+        roomId: 25,
+        winner: {
+          id: 1,
+          profileImage: 'https://item.kakaocdn.net/do/22b3b5f6c65114f383f5986c98828993616b58f7bf017e58d417ccb3283deeb3',
+          name: '남주영',
+        },
+      },
+      {
+        roomId: 24,
+        winner: {
+          id: 2,
+          profileImage: 'https://item.kakaocdn.net/do/22b3b5f6c65114f383f5986c98828993616b58f7bf017e58d417ccb3283deeb3',
+          name: '박건영',
+        },
+      },
+      {
+        roomId: 23,
+        winner: {
+          id: 3,
+          profileImage: 'https://item.kakaocdn.net/do/22b3b5f6c65114f383f5986c98828993616b58f7bf017e58d417ccb3283deeb3',
+          name: '서지수',
+        },
+      },
+      {
+        roomId: 22,
+        winner: {
+          id: 4,
+          profileImage: 'https://item.kakaocdn.net/do/22b3b5f6c65114f383f5986c98828993616b58f7bf017e58d417ccb3283deeb3',
+          name: '이준호',
+        },
+      },
+      {
+        roomId: 21,
+        winner: {
+          id: 5,
+          profileImage: 'https://item.kakaocdn.net/do/22b3b5f6c65114f383f5986c98828993616b58f7bf017e58d417ccb3283deeb3',
+          name: '솝트짱',
+        },
+      },
+    ],
+    hasNext: true,
+  };
+
+  return (
+    <WinnerBoard>
+      <WinnerHeader>👑 역대 우승자 명예의 전당 👑</WinnerHeader>
+      <WinnerList>
+        {WORDCHAIN_WINNERS_DATA.winners?.map(({ roomId, winner }) => {
+          const { id, profileImage, name } = winner;
```

Comment:
```suggestion
        {WORDCHAIN_WINNERS_DATA.winners?.map(({ roomId, winner: { id, profileImage, name }}) => {
```
p3; 구조분해할당 한다면 요렇게 해도 좋을 것 같습니다 !

p2; 그리고 `winners?.`에 옵셔널 체이닝 붙여준 이유가 있을까?!

Reply:
오호 좋아용!

옵셔널 체이닝을 붙여준 이유는 습관성..이긴 한데요! map을 돌렸을 때 로컬에서는 옵셔널체이닝을 붙이지 않아도 잘 나오던 데이터가 배포된 사이트에서는 undefined가 떴던 경험이 있어요..! 그 뒤로 습관처럼 map을 돌릴 때는 옵셔널 체이닝을 쓰고 있습니당..

Reply:
> 옵셔널 체이닝을 붙여준 이유는 습관성..이긴 한데요! map을 돌렸을 때 로컬에서는 옵셔널체이닝을 붙이지 않아도 잘 나오던 데이터가 배포된 사이트에서는 undefined가 떴던 경험이 있어요..! 그 뒤로 습관처럼 map을 돌릴 때는 옵셔널 체이닝을 쓰고 있습니당..

오호..?! 정확한 이유는 모르겠지만, 그럼 데이터가 정의해준 타입이랑 다르게 내려온 경우라서 어느 한쪽이 문제가 생긴 경우일거라고 생각해요. 혹은 typescript의 strict 옵션을 꺼서 사용했거나?! 

타입에 맞게 필요하다면 옵셔널체이닝을 자동으로 달아주거나 추천해주는데, 거기에 맞게 사용하는게 좋다고 생각합니다! 지금은 이어질 구문에서 useQuery로 데이터를 받아올거니까 옵셔널체이닝이 들어가는게 맞는데, 지금은 아니라고 생각해요!

Reply:
그리고 지금 createEndpoint 로 생성한 엔드포인트는 Response에 대한 타입 체킹을 하고 있어서 서버의 잘못된 타입이 컴포넌트까지 흘러들어올 일이 없어졌어요!

Reply:
이것이 타입스크립트...! 감사합니다!!!

Reply:
> 서버의 잘못된 타입이 컴포넌트까지 흘러들어올 일이 없어졌어요!

@Tekiter 없어졌다기 보다는 그러한 상황을 빨리 인지할 수 있게 된 거 아닌가요!!